charter/index.md

```diff
@@ -0,0 +1,228 @@
+possibly a W3C Working Group,
+under which members of this community should conduct technical work.
+
+* If, after producing such a charter,
+the Community Group determines that its current composition is adequate
+for conducting its work,
+then the charter produced is
+to be adopted as an updated Charter for this CG.
+
+* If the Community Group determines that it would be appropriate
+to divide the technical work into several venues,
+it may produce several such proposed charters
+rather than a single one.
+
+* Various supporting documents and articles
+describing the problem space
+may also be produced.
+
+All deliverables of this Community Group will use the
+[W3C Software and Document License](http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document).
+
+This group maintains an [index](https://font-text.github.io/work-items/)
+of the documents it is working on.
+
+## Community and Business Group Process and Patent Policy
+
+The group operates under the [Community and Business Group Process](https://www.w3.org/community/about/agreements/).
+Terms of in this charter that conflict with those of the Community and Business Group Process are void.
+
+As with other Community Groups,
+W3C seeks organizational licensing commitments
+under the [W3C Community Contributor License Agreement (CLA)](http://www.w3.org/community/about/agreements/cla/).
+(Technical Proposals in this Community Group charter are applicable “Specification” in the CLA).
+When people request to participate
+without representing their organization’s legal interests,
+W3C will in general approve those requests for this group
+with the following understanding:
+W3C will seek and expect an organizational commitment
+under the CLA starting with the individual’s first request
+to make a contribution to a group deliverable.
+The section on Contribution Mechanics describes
+how W3C expects to monitor these contribution requests.
+
+## Communication
+
+This group operates under the
+[W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/).
+
+This group will conduct its work publicly,
+primarily using the public-font-text@w3.org mailing list
+([archives](https://lists.w3.org/Archives/Public/public-font-text/))
+for minutes and announcements,
+and [GitHub](https://github.com/font-text)
+for discussions
+(See [Contribution Mechanics](#contribution-mechanics)).)
+
+Teleconferences and face-to-face meetings
+may be held from time to time,
+provided sufficient advance notice is given.
+Minutes of any such meeting will be
+posted to the group’s public mailing list
+in a timely manner,
+and will include
+a transcript of the meeting
+to the best of the minute-taker's ability,
+together with a list of any
+decisions and action items resolved within the meeting
+(See [Decision Policy](#decision-policy)).
+
+Deliverables and supporting documents
+produced by this Community Group are
+hosted on the group’s [GitHub](https://github.com/font-text).
```

Comment:
After we migrate to the [new repo](https://github.com/w3c/font-text-cg), this may need to be updated.

Reply:
Done.